patches/chrome-browser-resources-settings-privacy_page-cr_lottie.ts.patch

```diff
@@ -1,7 +1,22 @@
 diff --git a/chrome/browser/resources/settings/privacy_page/cr_lottie.ts b/chrome/browser/resources/settings/privacy_page/cr_lottie.ts
-index df65d69c2f134aa3954de2bdd7b355e37744fa1f..d3a4cdd2265268b69bae23f6684b26319d15ceef 100644
+index df65d69c2f134aa3954de2bdd7b355e37744fa1f..fce0231f8b1ae96114eff63dbe0c337c07f0be67 100644
 --- a/chrome/browser/resources/settings/privacy_page/cr_lottie.ts
 +++ b/chrome/browser/resources/settings/privacy_page/cr_lottie.ts
+@@ -89,10 +89,10 @@ export class CrLottieElement extends CrLitElement {
+     };
+   }
+ 
+-  animationUrl: string = '';
```

Comment:
why are we having to patch upstream for this if it's triggered by their own linter?

Reply:
cc: @fallaciousreasoning

Reply:
Create this so we can delete the patch after it lands:
https://github.com/brave/brave-core/pull/28548

Reply:
isn't there an ignore list for the linter we can add this to?

Reply:
Upstream has fixed in the following to CLs
https://chromium-review.googlesource.com/c/chromium/src/+/6266965
https://chromium-review.googlesource.com/c/chromium/src/+/6279072

```ts
  // their code after the above CLs
  accessor animationUrl: string = '';
  accessor autoplay: boolean = false;
  override accessor hidden: boolean = false;
  accessor singleLoop: boolean = false;

  // our code, after the patch:
  accessor animationUrl: string = '';
  accessor autoplay: boolean = false;
  override accessor hidden: boolean = false;
  accessor singleLoop: boolean = false;
```

so this patch should clean up on its own when we rebase :smile: